### DISCUSSION TITLE
Android TV Version (or optimization)

### DISCUSSION THREAD

**Discussion by vale44** [DATL4G/BurningSeries-Android#90] [Ideas]
Hello, I wanted to ask if an Android TV Version or Mode (like with other Apps on first open) are in plans. I have just installed the regular Apk on my Android TV and the interface looks already very usable, unfortunatly only remote control of the TV Box didn't interact with it properly and Navigation to play an episode or access the search bar was unfortunatly impossible, but some selection movement (while unintuitive) already worked, which is promising. I am no developer and can't judge for sure how difficult such an Integration would be, but from my novice perspektive it looks like not much is missing and it would be fantastic if we could use the App on Android tv because there is not many others with german sources!

**Comment by DatL4g** [comment]
Big change coming to TV soon(-ish) but navigation won't be part of it.

The new https://github.com/DatL4g/Mimasu app will have full Android TV support.
Issues being tracked here: https://github.com/DatL4g/Burning-Series/issues/84 and here https://github.com/DatL4g/Mimasu/issues/2

But feel free to suggest features etc here
